server/routes/permissions.js

```diff
@@ -51,15 +49,17 @@ router.get('/permissions/admins', co.wrap(function* (req, res) {
 
 
 router.post('/permissions/admins', co.wrap(function* (req, res) {
-    const userId = req.userDetails.email;
-    const admin = yield Admin.find({userId: userId});
-    if (!admin) {
-        return res.status(403).json([{"error": "action is not allowed"}]);
+    if (!permissionsUtils.isAdmin(req.userDetails)) {
```

Comment:
this should be accessible to admins only

Reply:
@may-ben-arie - it is, that's why you have this IF - if it's not the admin, it returns an error with 403 http code